packages/client/README.md

```diff
@@ -623,8 +934,8 @@ The library allows the user to choose a _partition key_, which simplifies publis
 
 The partition key can be given as an argument to the `publish` methods, and the library assigns a deterministic partition number automatically:
 
-```js
-await client.publish('my-stream-id', msg, Date.now(), msg.vehicleId)
+```typescript
+await client.publish(STREAM_ID, msg, Date.now(), msg.vehicleId)
```

Comment:
I would add to the above description an explicit note that `msg.vehicleId` is the partition key in this example.
```suggestion
await client.publish(STREAM_ID, msg, Date.now(), msg.vehicleId) // msg.vehicleId is the partition key here
```

Reply:
Agree with @harbu here